examples/database-with-web-server-and-redis/index.ts

```diff
@@ -0,0 +1,74 @@
+import { Database, Project, Services } from '@studion/infra-code-blocks';
+import * as aws from '@pulumi/aws';
+import * as awsx from '@pulumi/awsx';
+
+require('dotenv').config();
+
+const webServerImage = createWebServerImage();
+
+const dbName = process.env.DB_NAME || '';
+const dbUsername = process.env.DB_USERNAME || '';
+const dbPassword = process.env.DB_PASSWORD || '';
+
+const project: Project = new Project('database-project', {
+  services: [
+    {
+      type: 'DATABASE',
+      serviceName: 'database-example',
+      dbName: dbName,
+      username: dbUsername,
+      password: dbPassword,
+      applyImmediately: true,
+      skipFinalSnapshot: true,
+    },
+    {
+      type: 'REDIS',
+      serviceName: 'redis',
+      dbName: 'test-db',
+      region: 'us-east-1',
+    },
+    {
+      type: 'WEB_SERVER',
+      serviceName: 'web-server-example',
+      port: 3000,
+      image: webServerImage.imageUri,
+      desiredCount: 1,
+      size: 'small',
+      autoscaling: { enabled: false },
+      environment: (services: Services) => {
+        const db = services['database-example'] as Database;
+
+        const databaseConnectionString = db.instance.address.apply(
+          address =>
+            `postgres://${dbUsername}:${dbPassword}@${address}:5432/${dbName}`,
+        );
+        const redisConnectionString = process.env.REDIS_CONNECTION_STRING || '';
+
+        return [
+          {
+            name: 'DATABASE_CONNECTION_STRING',
+            value: databaseConnectionString,
+          },
+          {
+            name: 'REDIS_CONNECTION_STRING',
+            value: redisConnectionString,
+          },
```

Comment:
Same as with mongo example ⬆️ 
Please use secrets for database and redis passwords. Secrets can be obtained from db service passwordSecret property